packages/react/src/components/form/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { styles } from "./form.css";
+import { themeClass } from "../../theme/theme.css";
+import { useFlowState } from "./useFlowState";
+import { Initial } from "./initial";
+import { Authenticating } from "./authenticating";
+import { Error } from "./error";
+import { Success } from "./success";
+
+export const Form = () => {
+  const flowState = useFlowState();
+
+  return (
+    <div className={themeClass}>
+      <div className={styles.form}>Form</div>
+      <p>Flow state: {flowState.status}</p>
+      {flowState.status === "initial" && <Initial flowState={flowState} />}
```

Comment:
@vincenzoiozzo State based conditional rendering - makes it easy to figure out which UI should be rendered in the current state.